frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -905,32 +924,48 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
-                            <JSONEditorInput readOnly={readOnly} value={featureFlag.filters.payloads?.['true']} />
+                <div className="mb-6 flex flex-col gap-y-4">
+                    <div>
+                        <h3 className="l3">Payload</h3>
+                        {readOnly ? (
+                            featureFlag.filters.payloads?.['true'] ? (
+                                <JSONEditorInput readOnly={readOnly} value={featureFlag.filters.payloads?.['true']} />
+                            ) : (
+                                <span className="text-muted">No payload associated with this flag</span>
+                            )
                         ) : (
-                            <span>No payload associated with this flag</span>
-                        )
-                    ) : (
-                        <div className="w-1/2">
-                            <div className="text-muted mb-4">
-                                Specify a valid JSON payload to be returned when the served value is{' '}
-                                <strong>
-                                    <code>true</code>
-                                </strong>
+                            <div className="w-1/2">
+                                <div className="text-muted mb-4">
+                                    Specify a valid JSON payload to be returned when the served value is{' '}
+                                    <strong>
+                                        <code>true</code>
+                                    </strong>
+                                </div>
+                                <Group name={['filters', 'payloads']}>
+                                    <LemonField name="true">
+                                        <JSONEditorInput
+                                            readOnly={readOnly}
+                                            placeholder={'Examples: "A string", 2500, {"key": "value"}'}
+                                        />
+                                    </LemonField>
+                                </Group>
                             </div>
-                            <Group name={['filters', 'payloads']}>
-                                <LemonField name="true">
-                                    <JSONEditorInput
-                                        readOnly={readOnly}
-                                        placeholder={'Examples: "A string", 2500, {"key": "value"}'}
-                                    />
-                                </LemonField>
-                            </Group>
+                        )}
+                    </div>
+                    <div>
+                        <h3 className="l3">Recordings</h3>
+                        <p>Watch recordings of people who have been exposed to the feature flag.</p>
+                        <div className="inline-block">
+                            <LemonButton
```

Comment:
data attr so that we can track these?

could add https://posthog.com/docs/product-analytics/autocapture#capturing-additional-properties-in-autocapture-events so we can see which they click for multivariant

Reply:
Did it with a manual event because I had a named event for experiments and I want it to be the same for ease of finding